Azure-Samples/enterprise-data-sharing
# ydaponte/prerequirements

## Purpose
Automating pre-requirements and documenting them on ReadMe

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
Script automation and ReadMe documentation


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe: automating pre-requirements
```

## How to Test
*  Get the code

```bash
cd 1-data-marketplace/scripts
bash prerequirements.sh
```
Graph API permission need to be manually granted
and run main provision sh - should run without issues.
```bash
bash provision.sh
```
Closes issue #21
